backend/src/modules/users/dto/create-users.dto.ts

```diff
@@ -1,53 +1,39 @@
-import { Transform } from 'class-transformer';
-import { IsEmail, IsNumber, IsOptional, IsString, IsUUID, MaxLength, MinLength } from 'class-validator';
+import { IsEmail, IsNumber, IsOptional, IsString, IsUUID, Matches, MaxLength, MinLength } from 'class-validator';
 
 export class CreateUserDto {
   @IsString()
   @MaxLength(100, { message: 'First name must be less then 100 symbols' })
   @MinLength(2, { message: 'First name must be more then 2 symbols' })
-  @IsOptional()
   firstName!: string;
 
   @IsString()
   @MaxLength(100, { message: 'Last name must be less then 100 symbols' })
   @MinLength(2, { message: 'Last name must be more then 2 symbols' })
-  @IsOptional()
   lastName!: string;
 
-  @IsEmail()
+  @IsEmail({}, { message: 'Invalid email format' })
   @MaxLength(40)
-  @IsOptional()
   email!: string;
 
   @IsNumber()
-  @IsOptional()
   age!: number;
 
+  @IsString()
```

Comment:
In decorators important order of execution, they start from bottom to top, and in your case you should firstly check if it's string then check if min length equals to 6